File: LAB2/GCD.c
#include<stdio.h>

long int GCD(int a, int b)
{
    if(b == 0)
       return a;
    else 
       return GCD(b, a%b);   
}

int main()
{
    int num1,num2, result;
    printf("Enter a number1: ");
    scanf("%d", &num1);
    printf("Enter a number2: ");
    scanf("%d", &num2);
    result = GCD(num1,num2);
    printf("The GCD  of %d and %d is %d.\n", num1,num2,result);
    return 0;
}

File: 2.Recursion/GCD/GCD.py
def GCD(a, b):
    if b == 0:
        return a
    else:
        return GCD(b, a % b)

num1 = int(input("Enter a number1: "))
num2 = int(input("Enter a number2: "))
result = GCD(num1, num2)
print(f"The GCD of {num1} and {num2} is {result}.")